setup.py

```diff
@@ -60,35 +59,6 @@
     'cherrypy.scaffold',
     'cherrypy.wsgiserver',
 ]
-data_files = [
-    ('cherrypy', [
-        'cherrypy/cherryd',
-        'cherrypy/favicon.ico',
```

Comment:
Once again I've confirmed that favicon.ico is working, even for the tutorial:

```
$ python -m rwt 'git+https://github.com/cherrypy/cherrypy@issue-1538' -- ~/p/cherrypy/cherrypy/tutorial/tut01_helloworld.py &
[1] 95304
$ Collecting git+https://github.com/cherrypy/cherrypy@issue-1538
  Cloning https://github.com/cherrypy/cherrypy (to issue-1538) to /private/var/folders/c6/v7hnmq453xb6p2dbz1gqc6rr0000gn/T/pip-gdxxok4i-build
Collecting six (from CherryPy==8.5.1.dev6+ng9f8efb28)
  Using cached six-1.10.0-py2.py3-none-any.whl
Installing collected packages: six, CherryPy
  Running setup.py install for CherryPy ... done
Successfully installed CherryPy-8.5.1.dev6+ng9f8efb28 six-1.10.0
[27/Dec/2016:12:16:42] ENGINE Listening for SIGTERM.
[27/Dec/2016:12:16:42] ENGINE Listening for SIGHUP.
[27/Dec/2016:12:16:42] ENGINE Listening for SIGUSR1.
[27/Dec/2016:12:16:42] ENGINE Bus STARTING
[27/Dec/2016:12:16:42] ENGINE Started monitor thread '_TimeoutMonitor'.
[27/Dec/2016:12:16:42] ENGINE Started monitor thread 'Autoreloader'.
[27/Dec/2016:12:16:42] ENGINE Serving on http://127.0.0.1:8080
[27/Dec/2016:12:16:42] ENGINE Bus STARTED

$ curl -I http://localhost:8080/favicon.ico
127.0.0.1 - - [27/Dec/2016:12:16:54] "HEAD /favicon.ico HTTP/1.1" 200 1406 "" "curl/7.51.0"
HTTP/1.1 200 OK
Content-Type: image/x-icon
Server: CherryPy/8.5.1.dev6+ng9f8efb28
Date: Tue, 27 Dec 2016 17:16:54 GMT
Last-Modified: Tue, 27 Dec 2016 17:16:38 GMT
Accept-Ranges: bytes
Content-Length: 1406
```

If you're still getting a 404 (I haven't changed anything), can you dig deeper and identify where the file goes missing in your environment?